src/eval-frame/components/reps/rep-utils/rep-serialization-core-types.js

```diff
@@ -62,6 +63,7 @@ export class ValueSummary {
     this.size = size; // number
     this.stringValue = stringValue; // string
     this.isTruncated = isTruncated; // bool
+    this.DESC_TYPE = "ValueSummary";
```

Comment:
ignore all the changes like this in this file. just laying groundwork for later.